go.mod

```diff
@@ -288,11 +288,10 @@ replace (
 
 	// force all dependecies to use the same versions of ibc, tendermint and cosmos-sdk
 	github.com/cosmos/ibc-go/v3 => github.com/cosmos/ibc-go/v3 v3.4.0
-	github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.24
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.11
```

Comment:
i think we still want to keep this to ensure that all dependencies are using the same version of cosmos sdk (liquidity module etc)

Reply:
I think this got messed up when I brought it up to date with main